doc/lua_api.txt

```diff
@@ -2201,13 +2201,17 @@ appearing when you don't expect them to. See `no_prepend[]`
 
 #### `textarea[<X>,<Y>;<W>,<H>;<name>;<label>;<default>]`
 * Same as fields above, but with multi-line input
-* if the text overflows a vertical scrollbar is added
-* if the name is empty the textarea is readonly, the label is not displayed.
+* If the text overflows, a vertical scrollbar is added.
+* If the name is empty, the textarea is readonly and
+  the background is not shown, which corresponds to a multi-line label.
 
 #### `label[<X>,<Y>;<label>]`
 * `x` and `y` work as per field
-* `label` is the text on the label
-* Position and size units are inventory slots
+* The label formspec element displays the text set in `label`
+  at the specified position.
+* For text with multiple lines, refer to textarea.
+* To display text which can contain formspec code, e.g. text set by a player,
+  use `minetest.formspec_escape`.
```

Comment:
That’s not label-specific. As well as note on units you removed.

Reply:
Often people forget to use minetest.formspec_escape (see travelnet), so I mentioned it here.

When I removed the note on units I thought that somewhere it is mentioned that the note pertains on all size units.
This note does not make sense to me because it appears for some formspec elements and for others no information about the size units is given. Why is this?

Reply:
> it appears for some formspec elements and for others no information about the size units is given. Why is this?

[Historically](https://github.com/minetest/minetest/commit/3899f83da879e11cf531c88514772b890d060721), it was repeated in each element description. But later people probably considered that too obvious to repeat.

I suggest to move both size note and escaping explanation somewhere in the Formspec section itself, and keep short reminders like “Remember to escape text with formspec_escape” in elements documentation.